### PR TITLE
WEB: update web/pandas/versions.json for docs dropdown menu

### PR DESCRIPTION
**Pull request by simonjayhawkins**
I don't think the milestone is relevant for web updates so backport not needed?

### CODE REVIEW

**Comment by jreback** [comment]
merge when ready

**Comment by simonjayhawkins** [comment]
> merge when ready

does ci check the web build or just upload it?

**Comment by simonjayhawkins** [comment]
ahh yes. ci does build something so should wait for that to complete.

**Comment by jreback** [comment]
@meeseeksdev backport 1.4.x

**Comment by jreback** [comment]
prob don't need to backport but what-the-heck

**Comment by jreback** [comment]
oh guess backport didn't work. hmm

**Comment by jorisvandenbossche** [comment]
> I don't think the milestone is relevant for web updates so backport not needed?

Indeed, web things only are served from master, so never need to be backported.

File: web/pandas/versions.json [hunk 1]
```diff
@@ -4,13 +4,9 @@
         "version": "docs/dev"
     },
     {
-        "name": "1.4 (rc)",
+        "name": "1.4 (stable)",
         "version": "pandas-docs/version/1.4"
     },
-    {
-        "name": "1.3 (stable)",
-        "version": "docs"
```
Comment (jorisvandenbossche) [inline web/pandas/versions.json:12]: Small note that we should keep the entry with just the "docs" version (that's for the plain pandas.pydata.org/docs url), but with an updated version number -> https://github.com/pandas-dev/pandas/pull/45590

**Pull request merged**